css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,20 @@
+button {
+  font-size: 20px;
+  padding: 8px 22px;
+}
+
+button:hover {
+  cursor: pointer;
+  color: white;
+  background-color: black;
+}
+
+th {
+  color: white;
+  background-color: black;
+}
+
+.grey {
```

Comment:
while this does work, you want to complete this functionality using the `nth-child` selector.